packages/requisitions/src/RequestRequisition/DetailView/Toolbar/ToolbarActions.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import {
+  Autocomplete,
+  InputWithLabelRow,
+  useTranslation,
+  useConfirmationModal,
+} from '@openmsupply-client/common';
+import { useRequestFields, useIsRequestDisabled } from '../../api';
+
+const months = [1, 2, 3, 4, 5, 6];
```

Comment:
It seemed a little like the zepplin pic had this input as a text input. I went with a drop down with just these options.

Main reason: It was easier 

More info:
- It's easier to trigger a confirmation modal and update + refetch the lines rather than do it all client side
- When we add max months of stock changing, which will update the suggested quantities (which is also easier to just do through the API) it will be consistent
- Surely no one uses any other value other than 1-6.. right?!

can look at changing to a number input if you think that's better. Could trigger the update on debounce, or something